### DISCUSSION TITLE
call-flow diagram

### DISCUSSION THREAD

**Discussion by gblais** [tonsky/FiraCode#1478] [Q&A] [answered]
I use  | < - > characters to draw call-flow diagrams.
like this (courier font in gvim):
![2022-10-18 13_15_50- No Name  + - GVIM2](https://user-images.githubusercontent.com/47043980/196500154-5f665d96-b151-496e-bed5-a3ed7e12452b.png)

I'm using Fira Code (version 6.2) in Window Terminal (WTpreview version 1.16.2641.0) and the output is not what I would hope. The | characters get shifted when next to a long arrow and in one place (second hello) the starting | disappears completely:
![2022-10-18 13_05_36-New Discussion · tonsky_FiraCode — Mozilla Firefox](https://user-images.githubusercontent.com/47043980/196502304-60715639-2802-453b-a8ee-04fadd2fecdd.png)

In Windows Terminal settings the font is currently configured with the following features:
```
                "font":
                {
                    "face": "Fira Code",
                    "size": 10.0,
                    "weight": "normal",
                    "features":
                    {
                        "cv02": 1,
                        "cv25": 1,
                        "cv26": 1,
                        "cv28": 1,
                        "cv30": 1,
                        "cv32": 1,
                        "ss03": 1,
                        "ss04": 1,
                        "ss07": 1,
                        "ss09": 1
                    }
                }
```

Are there any settings I could use to make the call-flow diagram look better ?
Or should I open an issue? .. for the dropped | character for example.
Any help appreciated.

**Comment by tonsky** [comment, marked as answer]
Should be Windows Terminal problem. Works fine for me with all the options you use on macOS in VS Code

<img width="614" alt="Screenshot 2022-10-19 at 16 45 38" src="https://user-images.githubusercontent.com/285292/196724328-4250f6ed-9b61-4141-8361-508c8d6dedd8.png">
<img width="878" alt="Screenshot 2022-10-19 at 16 45 46" src="https://user-images.githubusercontent.com/285292/196724342-fe4398c8-c4e2-43db-aa51-9b08516e06c8.png">

**Reply by gblais** [reply]
Thanks for the reply. Yes you are correct. I tried in VS code (on windows) and it works fine there too.
As a side note, if anyone else has that problem with Windows Terminal, I found that changing the font size helped.
The problem for me happened with a font size of 10 or smaller, but with any larger size (e.g.: "size": 11.0) the problem seems to be gone.  A bit large for my taste, but at least things line up.
